src/main/java/frc/robot/lib/bindings/DriveBindings.kt

```diff
@@ -148,7 +182,7 @@ private fun CommandXboxController.configureReefAlignments() {
         .onFalse(
             ConditionalCommand(
                 SuperStructure.smartGoTo(RobotState.CoralStorage),
-                SuperStructure.smartGoTo(RobotState.NewCoralStation),
+                SuperStructure.smartGoTo(RobotState.CoralStorage),
```

Comment:
We played around with going to the 'stow' position after scoring. It drastically reduced robot movement. Might need an adjustment / amendment to smartGoTo that moves pivot and elevator, since it's mainly the wrist motion that causes issues. 

If we change all 'going down' movements to go to the storage position, we could change the logic in smartGoTo to move elevator and pivot together for those motions. Maybe it's better to move pivot and elevator, and then move wrist